### PR TITLE
created a new query to use inside charts

### PR DESCRIPTION
**Pull request by matheusgolemb**

### CODE REVIEW

File: src/articles/feed_copy.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+<?php
+if ($_SERVER['REQUEST_METHOD'] == "GET") {
+    $db = new dbServices($mysql_host, $mysql_username, $mysql_password, $mysql_database);
+    $dbConnected = $db->connect();
+    if ($dbConnected) {
+        $articles = $dbConnected->query("SELECT a.id,a.title,a.content_path,a.genre_id_01,a.genre_id_02,a.genre_id_03,a.likes,a.stores,a.datetime,u.first_name,u.last_name,u.email,g.genre FROM article_table a INNER JOIN user_table u ON u.id = a.user_id INNER JOIN genre_table g ON g.id = a.genre_id_01 ORDER BY `id` ASC");
+        $articles = $articles->fetch_all(MYSQLI_ASSOC);
+        $db->close();
+        if ($articles) {
+            echo(json_encode($articles));
```
Comment (gh-tyl) [inline src/articles/feed_copy.php:18]: Can you change your code like this?
But when you change here, you also need to change react code.

https://github.com/gh-tyl/wepost-api/blob/develop/src/articles/feed.php

```
            $res = array(
                "statusCode" => 200,
                "status" => "success",
                "data" => array(
                    "articles" => $articles
                )
            );
            echo json_encode($res
```

File: src/articles/feed_copy.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            exit();
+        } else {
+            echo "
+            {
+                \"statusCode\": 204,
+                \"status\": \"No Content\",
+                \"data\": {
+                    \"articles\": []
+                },
+            }";
```
Comment (gh-tyl) [inline src/articles/feed_copy.php:36]:
```
            $res = array(
                "statusCode" => 204,
                "status" => "No Content",
                "data" => array(
                    "articles" => []
                )
            );
            echo json_encode($res);
            exit();
```

File: src/articles/feed_copy.php [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        }
+    } else {
+        echo "
+        {
+            \"statusCode\": 500,
+            \"status\": \"Internal Server Error\",
+            \"data\": {
+                \"articles\": []
+            },
+        }";
```
Comment (gh-tyl) [inline src/articles/feed_copy.php:47]:
```
        $res = array(
            "statusCode" => 500,
            "status" => "Internal Server Error",
            "data" => array(
                "articles" => []
            )
        );
        echo json_encode($res);
```

File: src/articles/topics.php [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+<?php
+if ($_SERVER['REQUEST_METHOD'] == "GET") {
+    $db = new dbServices($mysql_host, $mysql_username, $mysql_password, $mysql_database);
+    $dbConnected = $db->connect();
+    if ($dbConnected) {
+        $topics = $dbConnected->query("SELECT g.genre, COUNT(a.genre_id_01) AS qtd FROM article_table a INNER JOIN genre_table g ON g.id = a.genre_id_01 GROUP BY g.genre ORDER BY qtd DESC LIMIT 5;");
+        $topics = $topics->fetch_all(MYSQLI_ASSOC);
+        $db->close();
+        if ($topics) {
+            echo(json_encode($topics));
```
Comment (gh-tyl) [inline src/articles/topics.php:18]:
```
            $res = array(
                "statusCode" => 200,
                "status" => "success",
                "data" => array(
                    "articles" => $articles
                )
            );
            echo json_encode($res)
```

File: src/articles/topics.php [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            exit();
+        } else {
+            echo "
+            {
+                \"statusCode\": 204,
+                \"status\": \"No Content\",
+                \"data\": {
+                    \"articles\": []
+                },
+            }";
```
Comment (gh-tyl) [inline src/articles/topics.php:36]:
```
            $res = array(
                "statusCode" => 204,
                "status" => "No Content",
                "data" => array(
                    "articles" => []
                )
            );
            echo json_encode($res);
            exit();
```

File: src/articles/topics.php [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+        }
+    } else {
+        echo "
+        {
+            \"statusCode\": 500,
+            \"status\": \"Internal Server Error\",
+            \"data\": {
+                \"articles\": []
+            },
+        }";
```
Comment (gh-tyl) [inline src/articles/topics.php:47]:
```
        $res = array(
            "statusCode" => 500,
            "status" => "Internal Server Error",
            "data" => array(
                "articles" => []
            )
        );
        echo json_encode($res);
```

**Review by gh-tyl — CHANGES_REQUESTED** [review]

**Comment by matheusgolemb** [comment]
I corrected the php code

**Review by gh-tyl — APPROVED** [review]
great job!

**Pull request merged**